ev3dev/display.py

```diff
@@ -179,14 +181,18 @@ def _map_fb_memory(fbfid, fix_info):
         )
 
 
-class Screen(FbMem):
+class Display(FbMem):
     """
     A convenience wrapper for the FbMem class.
     Provides drawing functions from the python imaging library (PIL).
     """
 
-    def __init__(self):
-        from PIL import Image, ImageDraw
+    GRID_COLUMNS = 22
+    GRID_COLUMN_PIXELS = 8
+    GRID_ROWS = 12
+    GRID_ROW_PIXELS = 10
+
+    def __init__(self, desc='Display'):
```

Comment:
I'm curious... what is the description for?

Reply:
It is just so if you print the object as a string the `__str__` has something to work with